src/core/utils/multicall.ts

```diff
@@ -9,11 +10,14 @@ interface PendingCall {
     allowFailure: boolean
   }
   resolve: (value: string) => void
-  reject: (reason: string | undefined) => void
+  reject: (reason: { message: string }) => void
```

Comment:
I didn't realise before, but this is changing the signature of reject. It'd be better if it was `reason?: any`.